docs/bokeh/source/docs/first_steps/first_steps_9.rst

```diff
@@ -106,23 +106,23 @@ Follow these steps to recreate the example from above:
    function to tell Bokeh which part of the Tuple to assign to either the start
    or the end of the plot's x-axis:
 
-    .. code-block:: python
+   .. code-block:: python
 
-        range_slider.js_link("value", p.x_range, "start", attr_selector=0)
-        range_slider.js_link("value", p.x_range, "end", attr_selector=1)
+       range_slider.js_link("value", p.x_range, "start", attr_selector=0)
+       range_slider.js_link("value", p.x_range, "end", attr_selector=1)
 
 8. Create a layout with all the elements of your dashboard and display it in a
    browser:
 
-    .. code-block:: python
+   .. code-block:: python
 
-        layout = layout([
-            [div, spinner],
-            [range_slider],
-            [p],
-        ])
+       layout = layout([
+           [div, spinner],
+           [range_slider],
+           [p],
+       ])
 
-        show(layout)
+    show(layout)
```

Comment:
The `show(layout)` needs to be part of the code block, it looks like it has been unindented outside of the code block here. 

But also are we sure these one-space indent changes are needed throughout? I thought the conclusion from the other PR was that the specific indent amount did not matter as long as it was consistent. Are all these occurrences of indent changes actually rendering incorrectly before?

Reply:
Thanks @bryevdv for the pointing me to this mistake.

We have to change something to avoid the appearance in [first_steps_9](https://docs.bokeh.org/en/latest/docs/first_steps/first_steps_9.html).

We could move the text after the counter one white space to the right or the code block one white space to the left. For me two white spaces after `1.` seems wrong and it would be something I would correct in a review.